client/landing/gutenboarding/onboarding-block/design-selector/index.tsx

```diff
@@ -82,10 +84,16 @@ const DesignSelector: React.FunctionComponent = () => {
 							} }
 						>
 							<span className="design-selector__image-frame">
-								<img alt={ design.title } src={ getDesignUrl( design ) } />
+								<img
+									alt=""
```

Comment:
I'd keep `alt={ design.title }`. It's useful when the image fails to load and especially better than `alt=""`. This looks like explicit empty alt text.

Reply:
This was suggested by https://github.com/Automattic/wp-calypso/pull/40745#discussion_r402901470  @simison

Reply:
Chrome is handling it OK 
![image](https://user-images.githubusercontent.com/17054134/78583272-e98b2980-7836-11ea-883b-187d36961db8.png)